src/qml/pages/MainPage.qml

```diff
@@ -52,7 +52,7 @@ Page {
 
                 Column {
                     Layout.fillWidth: true
-                    clip: true
+                    clip: false
```

Comment:
Clip has default value, remove this line

Reply:
It was TRUE previously